person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalizedecorator'
+require_relative 'trimmerdecorator'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
-  def initilize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] An `ArgumentError` occurs when the `Person` object is initialized. Please remove the `super` line highlighted above to get rid of this error.
![image](https://user-images.githubusercontent.com/46839548/216174680-c590b3a3-0415-418f-9b52-802b06483f20.png)
Please do the same for the `decorator.rb`. The app should run correctly when you do these 👍